middlewares/bodies.js

```diff
@@ -122,6 +127,7 @@ exports.createMember = async (req, res) => {
         body_id: req.currentBody.id
     });
 
+    // TODO: create active GSuite account
```

Comment:
This can also be done later

Reply:
Nevermind, this should create a GSuite account for new users. That should be implemented at the start already